docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -19,6 +19,31 @@ To start Promtail in dry run mode use the flag `--dry-run` as shown in the examp
 cat my.log | promtail --stdin --dry-run --client.url http://127.0.0.1:3100/loki/api/v1/push
 ```
 
+## Inspecting
+
+Promtail can be configured to print all changes to log entries as each pipeline stage is executed (via the flag `--inspect`).
+Each log entry contains four fields:
+- line
+- timestamp
+- labels
+- extracted fields
+
+Whenever any of these change, a log message will be printed:
+
+```bash
+cat my.log | promtail --stdin --dry-run --inspect --client.url http://127.0.0.1:3100/loki/api/v1/push
+```
+
+![inspect.png](inspect.png)
+
+It can be used in combination with `--stdin` and `--dry-run`.
+
+Additions will be highlighted in green, modifications in yellow, and removals in red.
```

Comment:
```suggestion
The output uses color to distinguish the modification. Additions are in green, modifications in yellow, and removals in red.
```

Reply:
I think this is more wordy and says the same thing TBH.

Reply:
You are correct.  My suggestion made the first sentence active (instead of passive) voice.  Active voice is the golden ticket that we strive for in our writing. . .

Reply:
Ah, right. Thanks for adding that detail.

I'm getting hung up on the use of "modification" in both sentences. I think I'll go for this if you have no objections:
"The output uses color to highlight changes. Additions are in green, modifications in yellow, and removals in red."